docs/usage/types.md

```diff
@@ -90,6 +94,9 @@ with custom properties and validation.
 `typing.FrozenSet`
 : see [Typing Iterables](#typing-iterables) below for more detail on parsing and validation
 
+`typing.Deque`
+: see [Typing Iterables](#typing-iterables) below for more detail on parsing and validation
```

Comment:
you mention "below" but I think you haven't added a deque case to example where these other tupes are demonstrated?

Reply:
Woops, fixed!